apps/nextjs-example/pages/index.tsx

```diff
@@ -340,6 +368,109 @@ function OptionalFunctionality() {
         );
     };
 
+    const onSubmitScript = async () => {
+        const payload = {
+            type: "script_payload",
+            code: {
+                bytecode: DO_NOTHING_SCRIPT
+            },
+            type_arguments: [],
+            arguments: []
+        };
+
+        const response = await signAndSubmitTransaction(payload);
```

Comment:
Wait so the only reason scripts don't work is because wallets don't handle the different `type`? 💀

Reply:
Yes, there's something weird that all wallets do, and inputs will likely have to be preserialized. I will look more into that later

Reply:
> Wait so the only reason scripts don't work is because wallets don't handle the different type?

Wait what do you mean by this?
Petra supports script payloads right?